newIDE/app/src/UI/Theme/DefaultTheme/Guidelines.css

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Yes that's the right approach, a CSS file with rules for the default-theme (`gd-guidelines-default-theme`) and dark theme in two separate files, and a third file with the common rules :)